src/Common/Reload.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import loadingFailure from '../Assets/Img/ic-loading-failure.png';
+
+export default function Reload({ reload = null, className = '' }) {
+    function refresh(e){
+        window.location.reload();
+    }
+    return (
+        <article className={`flex ${className}`} style={{ width: '100%', height: '100%' }}>
```

Comment:
instead of inline style can use w-100 h-100

Reply:
Done